### PR TITLE
More accurate docs about nanosecond timestamps.

### LINKED ISSUE #309: timestamps are rounded, not truncated
Update timestamp docs to specify that they are rounded to the nearest microsecond, not truncated.

### PR DESCRIPTION
**Pull request by mjibson**
Fixes #309

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/310)

### CODE REVIEW

File: timestamp.md [hunk 1]
```diff
@@ -22,7 +22,13 @@ When inserting into a `TIMESTAMP` column, use one of the following formats:
 
 Alternatively, you can cast a string as a timestamp: `CAST('2016-01-25T10:10:10' AS TIMESTAMP)`.
 
-Note that the fractional portion is optional and is truncated to microseconds (6 digits after decimal) for compatibility with the PostgreSQL wire protocol. To define or select a timestamp value at nanosecond resolution (9 digits after the decimal), you can use the `format_timestamp_ns(ts)` or `extract(nanosecond from ts)` string [functions](functions-and-operators.html).   
+Note that the fractional portion is optional and is rounded to
+microseconds (6 digits after decimal) for compatibility with the
+PostgreSQL wire protocol. To create a timestamp with nanoseconds, use the
+`parse_timestamp_ns(string)` [function](functions-and-operators.html)
+which will create a timestamp with nanoseconds from a string. To create
```
Comment (jseldess) [inline timestamp.md]: minor change: 
- "will create" to "creates"
- add a comma before "which"

File: timestamp.md [hunk 2]
```diff
@@ -22,7 +22,13 @@ When inserting into a `TIMESTAMP` column, use one of the following formats:
...
+a string with nanosecond precision, use `format_timestamp_ns(ts)`. The
+`extract` function supports nanoseconds: `extract(nanosecond from ts)`.
```
Comment (jseldess) [inline timestamp.md]: "supports nanoseconds as well" - for flow

**Comment by jseldess** [comment]
LGTM

**Comment by mjibson** [comment]
Done.

**Pull request merged**